build.gradle.kts

```diff
@@ -58,4 +63,29 @@ subprojects {
       }
     }
   }
+
+  tasks.withType<DokkaTaskPartial>().configureEach {
+    dokkaSourceSets.configureEach {
+      documentedVisibilities.set(
+        setOf(
+          DokkaConfiguration.Visibility.PUBLIC,
+          DokkaConfiguration.Visibility.PROTECTED
+        )
+      )
+
+      sourceLink {
+        val exampleDir = "https://github.com/TBD54566975/web5-sdk-kotlin/tree/main"
+
+        localDirectory.set(rootProject.projectDir)
+        remoteUrl.set(URL(exampleDir))
+        remoteLineSuffix.set("#L")
```

Comment:
eh?

Reply:
Have no idea, just copy pasted from the examples 🤷